style.css

```diff
@@ -730,11 +737,10 @@ input:focus:invalid {
   #proj-image {
     width: 684px;
     height: 448px;
-    background-color: blue;
   }
 
   #project-contents{
-    width: 80%;
+    width: 80%;   
   }
 
```

Comment:
- [ ] As requested by the previous reviewer you should stick to the design as much as possible. Please adjust the spacing and there should be a gap between each card. Please see the image below for reference
- [ ] Your design
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/692cb391-4320-4a20-accf-70d70903174e)

- [ ] Figma Design
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/6991e13e-7163-4fa7-9b5b-a8c15c884653)